platform/macos/scripts/deploy-packages.sh

```diff
@@ -114,6 +127,6 @@ if [[ "${GITHUB_RELEASE}" == true ]]; then
 fi
 
 buildPackageStyle "xpackage" "symbols"
-buildPackageStyle "xpackage SYMBOLS=NO"
+buildPackageStyle "xpackage SYMBOLS=NO" true
```

Comment:
I think it may benefit future readers if the “deploy the app, too” setting could be more easily understood on its face — could this be a named variable instead of a bool?

Reply:
Fixed in c7bf12e9be3df35f7992070b6c785e10664a7fe5 using a temporary environment variable, since Bash doesn’t otherwise have a natural way to name function arguments at the call site.